server/src/main/java/org/elasticsearch/cluster/metadata/Metadata.java

```diff
@@ -942,6 +947,15 @@ public Map<String, Custom> customs() {
         return this.customs;
     }
 
+    /**
```

Comment:
Here's an example in the related PR of how this will be used:

https://github.com/elastic/elasticsearch/pull/86224/files#diff-d017ebae07494c580da12a539a666fe1d87537f167e71761cf60451b8913d554R127

The related PR is out of date, it calls the API operatorState and it has the extra helper method to perform the additional .get.

Instead of:

```
OperatorMetadata existingMetadata = state.metadata().operatorState(namespace);
```

we'll be calling:

```
OperatorMetadata existingMetadata = state.metadata().operatorMetadata.get(namespace);
```